packages/react-scripts/template/README.md

```diff
@@ -442,27 +441,17 @@ Then we can change `start` and `build` scripts to include the CSS preprocessor c
    }
 ```
 
-Now running `npm start` and `npm run build` also builds Sass files. Note that `node-sass` seems to have an [issue recognizing newly created files on some systems](https://github.com/sass/node-sass/issues/1891) so you might need to restart the watcher when you create a file until it’s resolved.
+Now running `npm start` and `npm run build` also builds Sass files. Note that `node-sass-chokidar` seems to have an issue recognizing newly created files on some systems so you might need to restart the watcher when you create a file until it’s resolved.
```

Comment:
Well, what I did to test it was:
 1. Start a clean project and install node-sass-chokidar.
 1. run `node-sass-chokidar src/ -o src/ --watch --recursive`
 1. Create a new scss file `src/my_file.scss` with simple style `#yoyo {color:red;}` and save.
    - note that the file *HAS* to be in the directory that's being watched i.e. `src/`
 1. Check for the existence of `src/my_file.css`

that was all the "testing" I did but it works fine. So yeah, please describe your steps to reproduce and system info in an issue [here](https://github.com/michaelwayman/node-sass-chokidar/issues)